tests/scripts/thread-cert/thread_cert.py

```diff
@@ -235,6 +219,33 @@ def _setUp(self):
         self._inspector = debug.Inspector(self)
         self._collect_test_info_after_setup()
 
+    def _set_up_active_dataset(self, node, params, is_leader):
+        dataset = {
+            'master_key': binascii.hexlify(config.DEFAULT_MASTER_KEY).decode(),
+            'panid': params['panid'],
+        }
+
+        if is_leader:
```

Comment:
why do we need to know if the node is leader or not?

Can we setup the exact same active dataset on every node no matter it's a Leader node or not?

Reply:
Looks like it works, and is better. Updated.